README.md

```diff
@@ -84,24 +85,44 @@ Repository layout:
 	    ├── statics-dcg.yml                <- static routes eunetworks
 	    └── statics-eunetworks.yml         <- static routes DCG
 
+Dependencies:
+-------------
+
+    bgpq3 - http://snar.spb.ru/prog/bgpq3/
+    BIRD 1.6 - https://bird.network.cz/
+    OpenSSH client - https://www.openssh.com/
+    PHP - https://www.php.net/  # TODO: what's the minimum supported version?
+    Python 3 - https://www.python.org/  # TODO: what's the minimum supported version?
+    rsync - https://rsync.samba.org/
+
+You will also need the Python packages in `requirements.txt`.
+
+Most dependencies can be installed on Debian with `# apt install bgpq3 bird
+openssh-client php python3-jinja2 python3-numpy python3-requests python3-yaml
+rsync`. Some of the Python dependencies can't be found in the Debian
+repository. You can use [pip](https://pip.pypa.io/en/stable/) to install them:
+`$ pip3 install -r requirements.txt`.
+
 Usage:
 ------
 
-    ./generate-peer-config.sh - Generate new filters for peers, you need to run this before ./update-routers.sh push
-    ./update-routers.sh push  - To build a new config and push it to the routers, uses the filters generated by ./generate-peer-config.sh
-    ./update-routers.sh check - To build a new config and validate it, but don't push it to the routers
+Install the above dependencies first.
 
-Dependencies:
--------------
+Note that KEES currently requires the ability to write to the
+`/opt/routefilters/` and `/opt/router-staging/` directories. This is a bug, we
+are tracking it in [#15](https://github.com/coloclue/kees/issues/15).
 
-    rtrsub - https://github.com/job/rtrsub
-    jinja2 - jinja.pocoo.org
-    hiyapyco - https://pypi.python.org/pypi/HiYaPyCo
-    bgpq3  - https://github.com/snar/bgpq3
+    $ ./generate-peer-config.sh - Generate new filters for peers, you need to run this before ./update-routers.sh push
+    $ ./update-routers.sh push  - To build a new config and push it to the routers, uses the filters generated by ./generate-peer-config.sh
+    $ ./update-routers.sh check - To build a new config and validate it, but don't push it to the routers
+
+The `update-routers.sh` script expects to find `bird` on your PATH. Non-root
+users most likely want to run it like this:  
+`$ PATH="$PATH:/usr/sbin" ./update-routers.sh check`.
 
 Authors:
 -------
 
-Copyright (c) 2014-2017, Job Snijders <job@instituut.net>
-
-Copyright (c) 2017-2019, Network committee Coloclue <routers@coloclue.net>
```

Comment:
2020 as well? ;)

Reply:
Well I’m not on the networking committee, so I’m not changing that line...